src/rendering/renderers/gpu/pipeline/PipelineSystem.ts

```diff
@@ -277,24 +277,50 @@ export class PipelineSystem implements System
         {
             const attribute = geometry.attributes[attributeKeys[i]];
 
-            keyGen[index++] = attribute.location;
             keyGen[index++] = attribute.offset;
             keyGen[index++] = attribute.format;
             keyGen[index++] = attribute.stride;
+            keyGen[index++] = attribute.instance;
```

Comment:
Seems like it should be included since we use it in `_createVertexBufferLayouts`.